src/dom/context2d.mjs

```diff
@@ -1,10 +1,12 @@
-export default function(width, height, dpi) {
-  if (dpi == null) dpi = devicePixelRatio;
-  var canvas = document.createElement("canvas");
-  canvas.width = width * dpi;
-  canvas.height = height * dpi;
+export default function (width, height, options = {}) {
+  const {scale = devicePixelRatio, ...contextOptions} = !isNaN(options)
+    ? {...(options != null && {scale: options})}
+    : options;
```

Comment:
```suggestion
  const {scale = devicePixelRatio, ...contextOptions} = options && !isNaN(options)
    ? {scale: parseFloat(options)}
    : options;
```

Reply:
0 is currently working as before (in the sense that it returns a canvas of size 0)—it would be simpler if we didn't support this "feature".

Reply:
With the additional null check in the earlier comment we can remove `options &&` here.